lib/proxy/helpers.rb

```diff
@@ -25,7 +25,8 @@ def log_halt code=nil, exception=nil
 
   # read the HTTPS client certificate from the environment and extract its CN
   def https_cert_cn
-    certificate_raw = request.env['SSL_CLIENT_CERT'].to_s
+    ssl_client_cert_header = Proxy::SETTINGS.ssl_client_cert_header || 'SSL_CLIENT_CERT'
+    certificate_raw = request.env[ssl_client_cert_header].to_s.gsub(/   /, "\n")
```

Comment:
I tested `/[\t]+/` both on webrick and nginx and it's working properly. If you want this to be RFC compatible, how about `[ ]{2,}|\t+` to remove `two or more spaces` or `one or more tabs`? It's also working with both webrick and nginx and it's more bulletproof.